pkg/virt-controller/watch/vm/vm_test.go

```diff
@@ -7014,6 +7014,18 @@ var _ = Describe("VirtualMachine", func() {
 			})
 		})
 
+		Context("startVMI", func() {
+			It("should not start the VMI if the VM has the ManualRecoveryRequiredCondition set", func() {
+				vm := libvmi.NewVirtualMachine(libvmi.New(), libvmistatus.WithVMStatus(libvmistatus.NewVMStatus(libvmistatus.WithVMCondition(v1.VirtualMachineCondition{
+					Type:   v1.VirtualMachineManualRecoveryRequired,
+					Status: k8sv1.ConditionTrue,
+				}))))
+				vmCopy, err := controller.startVMI(vm)
+				Expect(err).To(BeNil())
+				Expect(vm).To(Equal(vmCopy))
+			})
+		})
+
```

Comment:
I would use the fakeclient (Note that I also added to ns to the VM, in order to make it works):
```suggestion
		Context("startVMI", func() {
			It("should not start the VMI if the VM has the ManualRecoveryRequiredCondition set", func() {
				vm := libvmi.NewVirtualMachine(libvmi.New(libvmi.WithNamespace(metav1.NamespaceDefault)), libvmistatus.WithVMStatus(libvmistatus.NewVMStatus(libvmistatus.WithVMCondition(v1.VirtualMachineCondition{
					Type:   v1.VirtualMachineManualRecoveryRequired,
					Status: k8sv1.ConditionTrue,
				}))))
				vmCopy, err := controller.startVMI(vm)
				Expect(err).To(BeNil())
				Expect(vm).To(Equal(vmCopy))
				vmiList, err := virtFakeClient.KubevirtV1().VirtualMachineInstances(metav1.NamespaceDefault).List(context.TODO(), metav1.ListOptions{})
				Expect(err).ToNot(HaveOccurred())
				Expect(vmiList.Items).To(BeEmpty())
			})
		})
```

Reply:
Done